web/src/store.js

```diff
@@ -25,6 +29,61 @@ export default new Vuex.Store({
     },
   },
   actions: {
+    async reloadApiKey({ state, commit, getters }) {
+      const { user } = getters;
+      const { status, data } = await state.girderRest.get(
+        'api_key', {
+          params: {
+            userId: user._id,
+            limit: 50,
+            sort: 'name',
+            sortdir: 1,
+          },
+        },
+      );
+
+      const [dandiKey] = data.filter(key => key.name === 'dandicli');
```

Comment:
In the case of 4XX and 5XX errors, this line will probably error before the status error check on like 46.

Data will be undefined or null, and this array destructure will throw `TypeError: data is not iterable`.

You probably want to move like 45 into the if block.  Ideally, you also want some error handling to display to the user.   You don't have to do it right now, but it might be good to stick a TODO nearby to do error handling.